packages/kbn-es-query/src/filters/build_filters/types.ts

```diff
@@ -59,9 +54,9 @@ export enum FilterStateStore {
 
 // eslint-disable-next-line
 export type FilterMeta = {
-  alias: string | null;
-  disabled: boolean;
-  negate: boolean;
+  alias?: string | null;
+  disabled?: boolean;
+  negate?: boolean;
```

Comment:
any chance we could fix the `params` below as well ? (to `SerializableRecord`)

Reply:
Lets do a separate PR, it was super hard to stabilize together